packages/admin-panel/src/VizBuilderApp/components/DataLibrary/AggregationDataLibrary.js

```diff
@@ -17,24 +17,43 @@ const aggregateToValue = aggregate =>
 
 const valueToAggregate = value => value.map(({ code, config }) => ({ type: code, config }));
 
-export const AggregationDataLibrary = ({ aggregate, onAggregateChange }) => {
+export const AggregationDataLibrary = ({ aggregate, onAggregateChange, onInvalidChange }) => {
   const [inputValue, setInputValue] = useState('');
-  const debouncedInputValue = useDebounce(inputValue, 200);
   const value = aggregateToValue(aggregate);
-  const { data: aggregationOptionSearchResults, isFetching } = useSearchAggregationOptions({
-    search: debouncedInputValue,
-  });
-  const options = inputValue ? aggregationOptionSearchResults : [];
```

Comment:
Remove `search` variable as we don't need it for autoComplete.

Reply:
We just pre-fetch all the aggregation options? Yup, that's a good plan 👍